src/ports/postgres/modules/utilities/path.py_in

```diff
@@ -40,7 +40,7 @@ m4_changequote(`<!', `!>')
 
 def path(schema_madlib, source_table, output_table, partition_expr,
          order_expr, pattern_expr, symbol_expr, agg_func,
-         persist_rows=None, **kwargs):
+         persist_rows=None, overlapping_patterns=None, **kwargs):
```

Comment:
I had received a similar comment on another module (sessionize), so thought it might be applicable here. @iyerr3 do you think setting the default to False is something we should do here too?